client/src/main/java/org/schemarepo/client/Avro1124RESTRepositoryClient.java

```diff
@@ -93,7 +93,7 @@ public Subject lookup(String subject) {
   public Iterable<Subject> subjects() {
     ArrayList<Subject> subjectList = new ArrayList<Subject>();
     try {
-      String subjects = webResource.get(String.class);
+      String subjects = webResource.accept(MediaType.TEXT_PLAIN_TYPE).get(String.class);
```

Comment:
I think there might be a simpler/cleaner way (just brainstorming here).

It would probably be possible to modify ```RepositoryServer``` so that it binds multiple ```RESTRepository``` instances to different root paths. This way, you could have ```localhost:2876/schema-repo``` serving just the machine-readable Content-Types (as it does currently, without this PR) and ```localhost:2876/schema-repo-browser``` serving the human-readable content. Both ```RESTRepository``` instances could be sharing 100% the same code, and be passed in the same ```Repository``` instance so that caching, state and synchronization is fully shared, but a different set of ```Renderer``` so that they can respond (and reject) requests appropriately.

I'm not sure exactly how to do this in Jersey, but there's got to be a way... It seems like a very typical use case.

Thoughts?